src/main/views/subscription-config-list-language.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "template.njk" %}
+{% from "macros/common-components.njk" import goBack, submitButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+    {{ super() }}
+    {{ goBack(
+        path= '/subscription-configure-list',
+        text = backButton,
+        clickEvent = "history.back();")
+    }}
+{% endblock %}
+
+{% block content %}
+    <div class="parent-box">
+        {% if noSelectionError %}
+            {{ govukErrorSummary({
+                titleText: error.title,
+                errorList: [
+                    {
+                        text: error.noSelection,
+                        href: "#english"
+                    }
+                ]
+            }) }}
+        {% endif %}
+        <form action="subscription-configure-list-confirmed" method="post">
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset" id="language-selector">
+                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                        <h1 class="govuk-fieldset__heading">
+                            {{ title }}
+                        </h1>
+                    </legend>
+                    <div class="govuk-radios" data-module="govuk-radios">
```

Comment:
If that not possible to use the nunjucks version of govukRadios rather that raw HTML here?

Reply:
fixed.